website/src/components/Changelog/GUI.tsx

```diff
@@ -26,6 +26,9 @@ export default function GUI({ title }: { title: string }) {
         <ChangeItem pull="6782">
           Adds always-on error reporting using sentry.io.
         </ChangeItem>
+        <ChangeItem enable={title === "Windows"} pull="6874">
+          Fixes a delay when closing the GUI
+        </ChangeItem>
```

Comment:
I couldn't replicate it on the Headless Client, this is explained by the Headless Client not having the IPC boundary, so it's just one thread and doesn't race between shutting down connlib and shutting down Sentry.

All I can think for the Linux GUI Client is that Linux shuts down a lot quicker, so there is a race but it never happened to trigger for me.